src/js/components/scroll.js

```diff
@@ -1,25 +1,30 @@
-/**
- * Scroll component. There is no need to create a new instance to it, since it won't listen to events,
- * nor anything dynamic.
- */
-import $ from 'jquery'
-import scroll from '../utils/scroll'
+import Jump from 'jump.js'
 
 const NAME = 'scroll'
-
 const DEFAULTS = {
   duration: 500,
   offset: -30
 }
 
-/* istanbul ignore next */
-$.fn[NAME] = function (options) {
-  options = options || {}
+class Scroll {
+  constructor (element, options) {
+    this.element = element
+    this.options = { ...DEFAULTS, ...options }
+  }
+
+  init () {
+    this.jumpElement()
```

Comment:
What about `buildElement`?

Reply:
I decided to use `jumpElement` because the `Jump.js` is instantiated